### PR TITLE
B 1517 handle hackathons tracks

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: app/h/[slug]/clients/track-show-more-project/track-show-more-project.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <>
+        {projects.map(project => (
+          <TrackProject project={project} key={project.name} />
+        ))}
+      </>
+    );
+  }
+
+  return (
+    <div className="flex w-full  flex-row items-center justify-center">
```
Comment (haydencleary) [inline app/h/[slug]/clients/track-show-more-project/track-show-more-project.tsx:30]:
```suggestion
    <div className="flex w-full flex-row items-center justify-center">
```

File: app/h/[slug]/mock.ts [hunk 2]
```diff
@@ -111,5 +111,164 @@ export const mock: components["schemas"]["HackathonsDetailsResponse"] = {
   sponsors: [],
   subtitle: "Contribute to the best open-source projects and get rewarded.",
   totalBudget: "$45,000",
-  tracks: [],
+  tracks: [
+    {
+      description:
+        "**Welcome all for this third edition of the ODHack set to start on April 22nd!**  \n" +
+        "\n" +
+        "🫡 Objective? Supporting the open-source journey by w**elcoming newcomers, giving opportunities to explore new projects & giving projects a boost looking for future contributions.**\n" +
```
Comment (haydencleary) [inline app/h/[slug]/mock.ts:119]:
```suggestion
        "🫡 Objective? Supporting the open-source journey by **welcoming newcomers, giving opportunities to explore new projects & giving projects a boost looking for future contributions.**\n" +
```

File: app/h/[slug]/mock.ts [hunk 3]
```diff
@@ -111,5 +111,164 @@ export const mock: components["schemas"]["HackathonsDetailsResponse"] = {
...
+        "\n" +
+        "🫶 Following exchanges with past contributors and project leads, we're introducing detailed guidelines to enhance your ODHack experience: **[Please find them here](https://www.notion.so/ODHack-Common-Guidelines-Project-leads-5e21836f85ad493f9ca429770af0ad40?pvs=21)**.  \n" +
+        "\n" +
+        "🔥Stay connected and updated by joining our [Telegram group](https://t.me/+Q1Oin_LMxpM0MTk8)! To facilitate better collaboration and communication, each project will also have its own dedicated Telegram group. \n" +
+        "\n" +
+        "🙌 Be fair and respectful please! No more than one issue assigned at a time to give everyone a chance. \n" +
+        "\n" +
+        "**How to join the ODHack**\n" +
+        "\n" +
+        "STEP 1 : Register on OnlyDust\n" +
```
Comment (haydencleary) [inline app/h/[slug]/mock.ts:131]:
```suggestion
        "Step 1: Register on OnlyDust\n" +
```

File: app/h/[slug]/mock.ts [hunk 4]
```diff
@@ -111,5 +111,164 @@ export const mock: components["schemas"]["HackathonsDetailsResponse"] = {
...
+        "\n" +
+        "**Telegram group** to stay posted, ask questions and so forth.\n" +
+        "\n" +
+        "Step 3: Pick your project\n" +
+        "\n" +
+        "Check out the projects participating in this edition to become familiar with it and when the D-day (April 22nd) is here, go, go, go!\n" +
+        "\n" +
+        "Step 4: Contribute (starting April 22nd)\n" +
+        "\n" +
+        "Select issues that match your skills, get assigned, and start making your mark! FIRST COME FIRST SERVE!\n" +
```
Comment (haydencleary) [inline app/h/[slug]/mock.ts:145]:
```suggestion
        "Select issues that match your skills, get assigned, and start making your mark! FIRST COME FIRST SERVED!\n" +
```

File: app/h/[slug]/mock.ts (hunk 2 above)
Comment (alexbeno) [reply app/h/[slug]/mock.ts:119]: it's a temporary mock don't need to but correct

File: app/h/[slug]/mock.ts (hunk 3 above)
Comment (alexbeno) [reply app/h/[slug]/mock.ts:131]: it's a temporary mock don't need to but correct

**Review by haydencleary — APPROVED** [review]

**Pull request merged**